src/it/scala/io/scalac/tezos/translator/LibrarySpec.scala

```diff
@@ -37,14 +40,29 @@ class LibrarySpec
       val testDb: MySQLProfile.backend.Database = DbTestBase.dbFromContainer(container)
 
       val libraryService = new LibraryService(libraryRepo, testDb)
-      val libraryRoute: Route = new LibraryRoutes(libraryService, system.log, config).routes
+      val userService = new UserService(new UserRepository, testDb) {
```

Comment:
why are you creating an alternative `UserSerivce` ? IMO in it tests we should stick to the `original` one

Reply:
To use the original one, we'd have to populate the database with a user, then call UserService.authenticateAndCreateToken and use this token to request. That would make the test way more complicated, and besides that our test case would be testing not only LibraryRoute and LibraryService, but also UserService, which wouldn't be clean.

Reply:
Im fine with that, since those are integration tests. This way we are testing local implementation of `UserService`. For now its "okish" since you are the one who wrote it and you can ensure that its not doing anything fishy.
The problem will arise when someone in the future will modify this test implementation because it will be convenient, or the "real" implementation will be changed (but test will stay the same), so tests and app will be using different logics.

in https://github.com/ScalaConsultants/Tezos-FullStack-Console-backend/pull/27/files i did add "real" `UserService` and adjusted the codebase, so each time user is added and bearer token is created.

Reply:
@piotr-urbanczyk will you change this to "prod" implementation ? ;)